### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by Rostabal** [XaF/TraktForVLC#140]
So I have installed the latest version of traktforvlc and activates it on the site. I have watched 3 movies today and none of them were tracked. I reinstalled vlc between each movie and the plugin as well. Nothing worked! What am I doing wrong?

**Comment by grenzionky** [comment]
me too....

**Comment by Rostabal** [comment]
Is this dead?

**Comment by xaf** [comment]
Not dead... Just don't have the time to find out the issues at the moment. Things are working fine on Linux as far as I know... Will investigate when I get the chance!

**Comment by Gliscor** [comment]
I installed this 3 days ago, the first time it worked, now it does not scrobble anything... I'm on Windows 7 Home Premium x64 fully updated, VLC 3.0.6 x64 and TraktForVLC_2.0.0a2.dev19.g7851368_windows_x64.exe.

**Comment by Gliscor** [comment]
Today I ran some test and here's the VLC log: 
[VLC log.log](https://github.com/XaF/TraktForVLC/files/2873390/VLC.log.log)

**Comment by GiacomoLaw** [comment]
@Gliscor getting the same issue. It seems not to start the worker when you start VLC up for the first time.

**Comment by glensc** [comment]
is there description of how this is supposed to even work? when using Kodi, first Kodi identifies the movie with imdb/tmdb, how to do that with vlc?

I can't find any menu item to invoke sync or scan or show imdb/tmdb info...

**Comment by glensc** [comment]
readme mentions third party libraries:
- https://github.com/XaF/TraktForVLC#external-libraries-and-resources

should I install those python modules? how?

**Comment by GiacomoLaw** [comment]
No, I believe those are installed with the plugin. Just doesn't seem to start after the first install.

**Comment by glensc** [comment]
I have never seen it "started", so I don't know what to expect.

**Comment by glensc** [comment]
show it appear under `Menu->Addons Manager`?

**Comment by GiacomoLaw** [comment]
https://github.com/iamkroot/trakt-scrobbler

Found this which is working well. Using it from now on.

Little more info on it: https://www.thenerdystudent.com/2019/06/trakt-scrobbler/

**Comment by xaf** [comment]
> Just doesn't seem to start after the first install.

This is a weird issue that I haven't been successful in reproducing in the little spare time I've had in the last few months / year. Can you specify what you mean by "start"? Are you talking about the lua add-on or the python tooling? If the second one, are you on Linux / windows / macos? Have you verified the service is running for Windows? What are the log messages reporting?

**Comment by xaf** [comment]
@glensc the installer is supposed to set things up for you, and already contains the required libraries. The list in the readme is for development purposes

**Comment by GiacomoLaw** [comment]
The process can't be found in processes when running a video on VLC. It simply doesn't show up.

**Comment by xaf** [comment]
@GiacomoLaw could you check your windows services? And see if a TraktForVLC service is there.

**Comment by GiacomoLaw** [comment]
Nope, it isn't - hasn't been since restart after initial install.

**Comment by glensc** [comment]
on a macOS:

- https://github.com/XaF/TraktForVLC/releases/tag/latest `Latest (updated on 2018-06-21 21:06 UTC)` - nothing happens
- https://github.com/XaF/TraktForVLC/releases/tag/v2.0.0a2 - `TraktForVLC 2.0.0a2 "Altered Carbon"` - works well. see trakt messages in vlc debug logs and "now playing information" on trakt.org and so on

the diff between those tags:
- https://github.com/XaF/TraktForVLC/compare/v2.0.0a2..7851368d7
